packages/nuxt/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare global {
+  const __NUXT_VERSION__: string
```

Comment:
We probably can introduce `window.$nuxt` back only with limited information `{ versions }` to reduce side-effects of nuxt on environment

Reply:
this file is only for typing development in this monorepo - it's not shipped to end user